recipes/voms/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eu
+
+export LIBS="-lz"
+./autogen.sh
+
+sed -i.bak "s#/usr/bin/soapcpp2#${PREFIX}/bin/soapcpp2#g" configure
+rm configure.bak
+
+./configure \
+    --prefix="${PREFIX}" \
+    --with-gsoap-wsdl2h="${PREFIX}/bin/wsdl2h"
+
+make -j1
+# make -j${CPU_COUNT}
```

Comment:
Just curious why this did not work. Also, do you have a `make check` here?

Reply:
The Makefile is buggy and doesn't declare the dependencies correctly. Fixing it isn't trivial and he build is fast enough with a single core so I gave up.

> Also, do you have a make check here?

No there isn't.